src/golang/lib/models/shared/engine.go

```diff
@@ -37,7 +37,7 @@ type AqueductCondaConfig struct {
 }
 
 type AirflowConfig struct {
-	ResourceID uuid.UUID `json:"integration_id"  yaml:"integration_id"`
```

Comment:
Is this migration going to be taken care of as part of PR 6?

Reply:
yep!

Reply:
@saurav-c actually I think we're gonna give up on renaming the fields themselves due to time constraints. Synced with @likawind and it's actually quite a lot of work with low benefit, so I'll just make a follow up task.